.github/workflows/continuous_integration.yml

```diff
@@ -13,12 +13,12 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.6", "3.7", "3.8"]
```

Comment:
Will this affect Python versions used in projects?

Reply:
This is what we're testing against, but we didn't change any of the code, so users running Python 3.6+ should be able to run this no problem, just as they are now.